TLSharp.Core/TelegramClient.cs

```diff
@@ -110,15 +112,15 @@ public bool IsUserAuthorized()
 
             var authCheckPhoneRequest = new TLRequestCheckPhone() { phone_number = phoneNumber };
             var completed = false;
-            while(!completed)
+            while (!completed)
```

Comment:
If you don't go through the pain of fixing your pull requests and always rely on me, you will never propose clean PRs ;)